src/test/java/com/google/api/codegen/testdata/java_settings_library.baseline

```diff
@@ -658,7 +701,7 @@ public class LibraryServiceSettings extends ServiceApiSettings {
 
       addLabelSettings = SimpleCallSettings.newBuilder(LabelerGrpc.METHOD_ADD_LABEL);
 
-      methodSettingsBuilders = ImmutableList.<ApiCallSettings.Builder>of(
+      unaryMethodSettingsBuilders = ImmutableList.<ApiCallSettings.Builder>of(
```

Comment:
Right, applyToAllApiMethods does not support streaming methods since right now we do not have any settings for streaming methods yet.